src/main/java/uk/gov/companieshouse/pscfiling/api/controller/FilingDataControllerImpl.java

```diff
@@ -4,13 +4,14 @@
 import javax.servlet.http.HttpServletRequest;
```

Comment:
I don't think we can replace the transaction look up in the FilingData Controller.  The request for this doesn't pass through the Transaction Interceptor where the transaction gets added to the request.
![Screenshot 2023-02-21 at 1 37 32 pm](https://user-images.githubusercontent.com/2736331/220360320-ddffbe3f-4e6b-4617-a3a9-f77231feff08.png)